gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/AppCdsSettings.kt

```diff
@@ -0,0 +1,202 @@
+package org.jetbrains.compose.desktop.application.dsl
```

Comment:
After merging the PR, please close the user PR https://github.com/JetBrains/compose-multiplatform/pull/2080